src/renderers/shared/stack/reconciler/ReactCompositeComponent.js

```diff
@@ -197,8 +196,11 @@ var ReactCompositeComponentMixin = {
 
     var Component = this._currentElement.type;
 
+    // Select the update queue associated to the transaction
```

Comment:
I think I will just delete it. We may want to use this mechanism for other uses.